ecs-cli/modules/cli/cluster/cluster_app.go

```diff
@@ -11,7 +11,7 @@
 // express or implied. See the License for the specific language governing
 // permissions and limitations under the License.
 
-package command
+package cluster
```

Comment:
I like the second one more, it's easier to see what file you're editing during development.